lib/pymedphys/_experimental/pinnacle/rtstruct.py

```diff
@@ -296,7 +299,18 @@ def read_roi(ds, plan):
                     points[1] = round(points[1], 5)
                     points[2] = round(points[2], 5)
 
+                curr_points[0] = curr_points[0] \
+                    - (1*plan.ct_center[0]) \
```

Comment:
A test is failing at this point with "IndexError: list index out of range" indicating that ct_center is empty at least for the test dataset.

Would we always expect there to be a ct_center point?